### ISSUE TITLE
OCP4.6 - Rebuild lldpad Ops Tool

### ISSUE THREAD

**Issue by StevenBarre** [BCDevOps/OpenShift4-RollOut#443]
**Describe the issue**
The [lldpad](https://github.com/bcgov-c/platform-tools/tree/ocp4-base/ocp4/lldpad) image is built on UBI8 and should be rebuilt as part of quarterly patching to pull in the latest security patches.

**Which Sprint Goal is this issue related to?**

**Additional context**

**Definition of done Checklist (where applicable)**
- [x] Rebuild the lldpad container image
- [x] Deploy to LAB and test
- [ ] Deploy to SILVER

**Comment by StevenBarre** [comment]
Blocked on the 2021Q2 Silver upgrade CHG

**Comment by StevenBarre** [comment]
Not cooperating on Silver reliably. Needs investigation.

**Comment by wmhutchison** [comment]
May or may not even keep lldpad for the long term since it doesn't seem DXC Network folks are making use of this, and it's for their sake that we're attempting to make this work in Openshift.

**Comment by wmhutchison** [comment]
Closing this off, we're yanking lldpad from all OCP4 clusters for now due to too many technical issues and zero input/feedback from Network. May revisit at a later time/date. Will create a separate ticket to track disabling of lldapd pods on remaining clusters.
